lib/potluck.rb

```diff
@@ -0,0 +1,35 @@
+class Potluck
+  attr_reader :date,
+              :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu_items = Hash.new([0])
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category)
+    @dishes.find_all do |dish|
+      if dish.category == category
+        dish
+      end
+    end
+  end
+
+  def menu
+    menu_items = Hash.new(0)
```

Comment:
I think at some point you wanted to make a Hash with a default value of an empty array, but you probably found out that Hash.new([]) doesn't work. If you google how to do this, you'll see that it is pretty complicated. I would encourage you to look it up and figure out how it works!

Reply:
Looks like you put in a really good effort here.